test/MUIDataTableFilter.test.js

```diff
@@ -18,7 +18,30 @@ describe('<TableFilter />', function() {
     columns = [
       { name: 'firstName', label: 'First Name', display: true, sort: true, filter: true, sortDirection: 'desc' },
       { name: 'company', label: 'Company', display: true, sort: true, filter: true, sortDirection: 'desc' },
-      { name: 'city', label: 'City Label', display: true, sort: true, filter: true, sortDirection: 'desc' },
+      {
+        name: 'city',
+        label: 'City Label',
+        display: true,
+        sort: true,
+        filter: true,
+        sortDirection: 'desc',
+        options: {
+          filter: true,
+          filterType: 'custom',
+          customFilterListRender: v => `City: ${v}`,
+          filterOptions: {
+            names: [],
+            logic(city, filters) {
+              return false;
+            },
+            display: (filterList, onChange, index, column) => (
```

Comment:
This is a good start, but was there a plan to provide an actual test case for this option? If not, I can work on one once this PR is merged.

Reply:
Would you terribly mind writing the test? Or giving some feedback on what you may want in the test?

Reply:
No problem, I tend to add what I can after merges but before releases. They're not required for the PRs, but are nice to have. Once the next release is up, you should be able to find the commit for it in the project if you want to look at it. Not sure myself how I'll tackle it yet!

Reply:
FYI, here is the PR with the test, in case you were still interested: https://github.com/gregnb/mui-datatables/pull/736.

Reply:
Oh, quite simple really! Thanks for adding that :)

Reply:
Yep, this one wasn't too bad, just needed to test that the display actually worked.